src/main/webapp/app/exercises/programming/assess/programming-assessment-dashboard/programming-exercise-submissions.component.ts

```diff
@@ -37,6 +38,7 @@ export class ProgrammingExerciseSubmissionsComponent implements OnInit {
     newManualResultAllowed: boolean;
     automaticType = AssessmentType.AUTOMATIC;
     private cancelConfirmationText: string;
+    assessedSubmissions: number;
```

Comment:
```suggestion
    numberAssessedSubmissions: number;
```

Reply:
Fix it in all kinds of exercises in d4596af691f73c5b338fa945cf0584bd614ffeec